openflow13/nx_action.go

```diff
@@ -929,18 +983,18 @@ func (a *NXActionDecTTL) UnmarshalBinary(data []byte) error {
 	err := a.NXActionHeader.UnmarshalBinary(data[n:])
 	n += int(a.NXActionHeader.Len())
 	if len(data) < int(a.Len()) {
-		return errors.New("the byte array has wrong size to unmarshal an NXActionDecTTL message")
+		return errors.New("the []byte is too short to unmarshal a full NXActionDecTTL message")
 	}
 	a.controllers = binary.BigEndian.Uint16(data[n:])
 	n += 2
-	a.zeros = []uint8{0, 0, 0, 0}
+	a.zeros = [4]uint8{}
 	return err
 }
 
 func NewNXActionDecTTL() *NXActionDecTTL {
 	a := &NXActionDecTTL{
 		NXActionHeader: NewNxActionHeader(NXAST_DEC_TTL),
-		zeros:          []uint8{0, 0, 0, 0},
+		zeros:          [4]uint8{},
```

Comment:
I don't think this needed

Reply:
Do you mean field zeros is not needed? If so, I'm afraid the message encoding might not be as expected.

Reply:
That's just me nitpicking. I mean that the explicit initialization is not needed, since `zeros` is now a fixed size array. By default all elements of the array will be initialized to 0